lib/BackgroundJob/BackgroundService.php

```diff
@@ -89,18 +89,33 @@ public function execute(int $timeout, bool $verbose, IUser $user = null, int $ma
 
 		// Here we are defining all the tasks that will get executed.
 		//
-		$task_classes = [
-			CheckRequirementsTask::class,
-			CheckCronTask::class,
-			LockTask::class,
-			DisabledUserRemovalTask::class,
-			StaleImagesRemovalTask::class,
-			CreateClustersTask::class,
-			AddMissingImagesTask::class,
-			EnumerateImagesMissingFacesTask::class,
-			ImageProcessingTask::class,
-			UnlockTask::class
-		];
+		if ($timeout > 0) {
```

Comment:
Not sure if this should be coupled with `timeout` or there should be another argument, something like `--favor-create` (this is stupid, but not sure what is better name here anyway)...

Reply:
Maybe `--clustering-last` :thinking:

Reply:
`--defer-clustering` maybe?  I like that a little better since it seems more like a command ("my intentions are to _defer clustering_") but maybe that's just me. And not that it really matters anyway 😅

Reply:
Great name.. Do you want to do it?.. 😬 
Now I add some comments that could help you. 😉